docs/source/use-bots.md

```diff
@@ -8,6 +8,36 @@ An example of the Kebechet bot in action can be seen below.
 <img alt="Kebechet Automatic Update" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/KebechetAutomaticUpdate.png">
 </div>
 
+## Set up Thoth bots on your fork
+
+Adding Thoth's bots takes just a few steps! Start by installing the Kebechet GitHub application by [following this link](https://github.com/apps/khebhut). It can be configured on an organization or on a single repository. To test out the Kebechet bot in this tutorial, select your profile for the organization, and specify the `elyra-aidevsecops-tutorial` as the repository.
+
+Once the application is installed, you will need to add Thoth's bots as collaborators. Navigate to your fork of `elyra-aidevsecops-tutorial`. Under the repository's **Settings**, go to **Manage Access** and click on "Invite a collaborator" and add Sesheta. Sesheta is a friendly Thoth bot who is used to help automate tasks. You might have already done this step when [building an image](build-images.md), in which case you do not have to invite Sesheta again. Please note: there is sometimes a delay in Sesheta's invite acceptance. Please note: Sesheta's invite acceptance is not instantaneous. [Follow this link](https://github.com/AICoE/aicoe-ci/issues/new?assignees=goern%2Charshad16&labels=area%2Fcyborgs%2Cbot%2Csig%2Fcyborgs&template=request_sesheta.yaml&title=Help+with+Sesheta+invite) and fill out the form to have Sesheta accept your invitation.
```

Comment:
```suggestion
Once the application is installed, you will need to add Thoth's bots as collaborators. Navigate to your fork of `elyra-aidevsecops-tutorial`. Under the repository's **Settings**, go to **Manage Access** and click on "Invite a collaborator" and add Sesheta. Sesheta is a friendly Thoth bot who is used to help automate tasks. You might have already done this step when [building an image](build-images.md), in which case you do not have to invite Sesheta again. Please note: currently Sesheta does not accept invites instantaneously and automatically (see [this issue](https://github.com/AICoE/aicoe-ci/issues/126) for more details). [Follow this link](https://github.com/AICoE/aicoe-ci/issues/new?assignees=goern%2Charshad16&labels=area%2Fcyborgs%2Cbot%2Csig%2Fcyborgs&template=request_sesheta.yaml&title=Help+with+Sesheta+invite) and fill out the form to have Sesheta accept your invitation.
```
Could we explicitly mention that invite won't be accepted automatically?

Reply:
yep-- done!